src/commands/interactive/sortTenantData.ts

```diff
@@ -0,0 +1,52 @@
+import { MeshTenant } from "../../mesh/mesh-tenant.model.ts";
+import { CmdGlobalOptions } from "../cmd-options.ts";
+
+export function sortTenantDataByCost(
+  _options: CmdGlobalOptions,
+  data: MeshTenant[],
+) {
+  data.sort(function (a: MeshTenant, b: MeshTenant) {
+    let costA = 0;
+    let costB = 0;
+
+    if (a != undefined) {
+      for (const cost of a.costs) {
+        if (cost.cost != undefined && cost.cost != "") {
+          costA += Number(cost.cost);
+        }
+      }
+    }
+    if (b != undefined) {
+      for (const cost of b.costs) {
+        if (cost.cost != undefined && cost.cost != "") {
+          costB += Number(cost.cost);
+        }
+      }
+    }
+
+    if (costA > costB) return -1;
+    else if (costB > costA) return 1;
+
+    return 0;
+  });
+
+  return data;
+}
+
+export function sortTenantDataByName(
+  _options: CmdGlobalOptions,
+  data: MeshTenant[],
+) {
+  data = data.sort(function (a, b) {
```

Comment:
In recent versions of javascript its preffered to use the arrow function lambda syntax. Because it is a bit more "safe" as the this pointer is handled better there (the this inside a function always points to the caller that can be confusing, while the this inside a arrow function is always scoped to the function itself. As you dont use this here, this makes not difference). The modern way would be:
```
data.sort((a, b) => {
// ...
});
```